FAQ.md

```diff
@@ -58,112 +50,79 @@ types of programming languages.
 
-The terms and conditions of standard support is available [here](https://slint-ui.com/support/slint_support_service_agreement) and premium support is available [here](https://slint-ui.com/support/slint_premium_support_service_agreement).
-
-### Ambassador license
-
-#### Why is the Ambassador license free-of-charge?
-
-The license is provided free-of-charge to achieve the following goals:
-
-a. accelerate adoption of Slint,
-b. allow developers to use Slint under a non-GPL license,
-c. create a strong feedback loop to improve Slint.
-
-#### When does the Ambassador license run out?
-
-The license grant is [perpetual](#what-does-perpetual-mean), which means that you can use Slint free-of-charge forever. The perpetual nature of the license also implies that even if we need to modify the terms of the license in the future, the modified terms will not apply to already granted licenses.
-
-#### For how long do you plan to offer the free-of-charge Ambasssador license?
-
-Forever. However, we may modify the terms of the license in the future based on user feedback and business needs.
+Refer to GPL FAQ [https://www.gnu.org/licenses/gpl-faq.en.html#LinkingWithGPL](https://www.gnu.org/licenses/gpl-faq.en.html#LinkingWithGPL).
 
-#### How can I get the Ambassador license?
+#### My MIT-licensed program links to Slint GPLv3. Can someone fork my program to build and distribute a proprietary program?
 
-The license is automatically granted on signing the license agreement [here](https://slint-ui.com/ambassador-program.html#application). The authorized signatory could be the code owner, primary maintainer, or in case of an organisation, the relevant person authorized to sign contracts on behalf of the organisation.
+Yes, provided the person distributing the proprietary program acquired a Slint proprietary license, such as the Slint Royalty-free license or a paid license, instead of using Slint under GPLv3. The other option would be to remove the dependency to Slint altogether.
 
-#### Do all contributors to my code have to sign up for the Ambassador license?
+#### My MIT-licensed program links to Slint GPLv3. How can I convey to someone that they can distribute my program as part of a proprietary licensed program?
 
-No. All contributions to the code is covered under the Ambassador license. Hence, contributors can use Slint under the same license within this scope.
+You can add a note as part of your license that to distribute a proprietary licensed program, one can acquire a Slint proprietary license or the dependency to Slint should be removed.
 
-### Distributions
+#### My MIT-licensed program links to Slint GPLv3. Under what license can I release the binary of my program?
 
-#### Do I need to pay to distribute my application on desktop?
+While your software modules can remain under the MIT-license, the work as a whole must be licensed under the GPL.
+Hence, the binary must be made available under the GPLv3.
```

Comment:
This seems to be a duplicate of "If I link my program with Slint GPLv3, does it mean that I have to license my program under the GPLv3, too?"

In that FAQ entry we write that merely GPL compatibility is required, while here we say that the "work as a whole must be licensed under the GPL". That seems somewhat contradicting to me.

I suggest to either remove the second FAQ entry (this one here) or align them (mention GPL compatibility for both).

Reply:
GPL license text distinguishes between program (referring to your own code) and the "work as a whole" (referring to the complete software that may include 3rd party libs). updated the question to reflect that wording.